AutoMLOps/AutoMLOps.py

```diff
@@ -307,3 +312,9 @@ def pipeline(bq_table: str,
             func=func,
             name=name,
             description=description)
+
+def clear_cache():
```

Comment:
Shouldn't this be in utils?

Reply:
I thought about putting it in utils but it's a function that the user calls directly; it's probably good practice to keep functions that the user calls directly together in the same file (e.g. generate, run, go, clear_cache, etc.)